config/production/airflow-vpc.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: "{{stack_group_config.aws_infra_templates_root_url}}/v0.3.0/templates/VPC/vpc-20-private.yaml"
```

Comment:
This should use the latest tag on the [aws-infra](https://github.com/Sage-Bionetworks/aws-infra/tags) repository. In this case, it's `v0.6.8`.

Reply:
changed in latest commit